docs/shared/recipes/nx-release/independent-releases.md

```diff
@@ -0,0 +1,77 @@
+# Releasing Projects Independently
+
+Nx Release supports releasing projects independently. This is useful when you have a monorepo with projects that are not released on the same schedule.
+
+## Configuring Independent Releases
+
+To configure independent releases, add the following property to your `nx.json` file:
+
+```nx.json
+{
+  "release": {
+    "projectsRelationship": "independent"
+  }
+}
+```
+
+## Differences from Fixed Releases
+
+Nx release will behave differently when configured for independent releases.
+
+### Prompt for Multiple Version Bumps
+
+When configured for independent releases, Nx Release will prompt for a version bump for each project that is being released. This allows the version of each package differ over time.
+
+### Create a Git Tag for Each Project
+
+Since each project can have a different version, Nx Release will create a git tag for each project that is being released. By default, the tag for each package will follow the pattern `{projectName}@{version}`. For example, if the `pkg-1` project is being released with version `1.1.0`, its git tag will be `pkg-1@1.1.0`.
+
+This can still be changed with the `release.releaseTagPattern` property in `nx.json`, but be sure to include `{projectName}` in the pattern.
```

Comment:
```suggestion
This can still be changed with the `release.releaseTagPattern` property in `nx.json`, but be sure to include `{projectName}` in the pattern so that each generated tag is unique.
```